packages/auth/src/commands/auth/token.ts

```diff
@@ -1,13 +1,14 @@
 import color from '@heroku-cli/color'
 import {Command, flags} from '@heroku-cli/command'
 import * as Heroku from '@heroku-cli/schema'
-import * as formatRelative from 'date-fns/formatRelative'
+import {Interfaces} from '@oclif/core'
+import formatRelative from 'date-fns/formatRelative'
 
 export default class AuthToken extends Command {
   static description = `outputs current CLI authentication token.
 By default, the CLI auth token is only valid for 1 year. To generate a long-lived token, use heroku authorizations:create`
 
-  static flags = {
+  static flags: Interfaces.FlagInput = {
```

Comment:
Not sure we need this with the additional command specified in package.json for -h. otoh, it probably won't hurt anything to leave as is